lib/src/task.rs

```diff
@@ -9,9 +9,25 @@ use cap_std::fs::Dir;
 use cap_std_ext::cap_std;
 use cap_std_ext::prelude::CapStdExtCommandExt;
 
+/// How much information we output
+enum CmdVerbosity {
+    /// Nothing is output by default
+    Quiet,
+    /// Only the task description is output
+    Description,
+    /// The task description and the full command line are output
+    Verbose,
+}
+
+impl Default for CmdVerbosity {
```

Comment:
Instead of manual `impl Default` here, better would be to derive it on the enum and add the `#[default]` attribute on `Description` - https://doc.rust-lang.org/std/default/trait.Default.html#enums

Reply:
Huh wait did that always exist?  I had a feeling it was relatively recent and maybe that's why I'd internalized the manual impl.  Anyways done and changed one other place!

Reply:
Looks like [since 1.56](https://github.com/rust-lang/rust/pull/86735/commits/c70147fd66e08962ab06adf12eb6a41bc1ea7f08)